src/lib/components/form/Form.component.tsx

```diff
@@ -264,7 +264,7 @@ const PageForm = forwardRef<HTMLFormElement, PageFormProps>(
     const requireMode = useContext(RequireModeContext);
     return (
       <ScrollbarWrapper>
-        <PageFormWrapper {...formProps} ref={ref}>
+        <PageFormWrapper {...formProps} noValidate ref={ref}>
```

Comment:
It should probably also be set on the TabForm. Also as we enforce it at this level, it should not be allowed by the Form props type (aka `Omit<FormProps, 'novalidate'>` is necessary)